src/components/navBar/navBar.stories.tsx

```diff
@@ -1,20 +1,59 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
+import { useTheme } from '@mui/material'
+import Avatar from '@mui/material/Avatar'
+import IconButton from '@mui/material/IconButton'
 import Typography from '@mui/material/Typography'
 import { Box } from '@mui/system'
+import type { Meta, StoryFn } from '@storybook/react'
 import React from 'react'
 
 import Icon from '../icon'
 import NavBar from './navBar'
 
-const meta = {
+const meta: Meta<React.ComponentProps<typeof NavBar>> = {
   title: 'Rustic UI/Nav Bar/Nav Bar',
   component: NavBar,
   tags: ['autodocs'],
   parameters: {
     docs: {
       description: {
         component:
-          'The `NavBar` component provides a customizable navigation bar that can be used to create a consistent layout across web applications. It typically includes a logo or branding element in the center, along with buttons or icons for accessing various navigation options such as drawers or menus. Note: The icon buttons will only be shown on mobile and tablet screens.',
+          'The `NavBar` component provides a customizable navigation bar for web applications, offering both top and bottom navigation options. The top navigation bar can include a logo and various navigation items, while the bottom navigation bar is optimized for mobile devices.',
```

Comment:
Shiti suggested to add component description in the `.tsx` file. Maybe we should keep the location of component description consistent